buildstockbatch/schemas/v0.3.yaml

```diff
@@ -53,7 +53,7 @@ hpc-postprocessing-spec:
 
 
 sampler-spec:
-  type: str(required=True)
+  type: enum('residential_quota', 'residential_quota_downselect', 'precomputed', 'commercial_sobol', required=True)
```

Comment:
I'd rather _not_ do this. When someone goes to add a sampler, they'll have to remember to update this. I'll add a validation function that checks that the sampler actually exists instead.